ui/icon-set/svg-material-icons.js

```diff
@@ -6,10 +6,33 @@
  */
 
 
+import {
```

Comment:
Seems like there is some problem with the generation tool, here outlined SVG imports doesn't look normal

Reply:
This is a manual process AFAIK. Probably some wrong renaming while doing a bulk update. Generated exports are correct.

Reply:
Please ignore my comment above. There is a huge banner at the top which says it's auto-generated, but yet I say it's manual 🤣

Reply:
It's not manual. There is a regex for finding the name and converting it. On the initial try, material-design was used for outlined, round and sharp. I had to move those three above the normal mat, because the mat once is kind of a catch-all.
Since there is an issue in this area, I'd appreciate some help with this. I will discuss on Discord if I can't figure it out.

Reply:
fixed with @yusufkandemir 's help